configure.ac

```diff
@@ -333,6 +333,7 @@ AC_ARG_WITH(docbook,
             AC_HELP_STRING([--with-docbook=FILE],
                            [compiling manpages using docbook in the specified path, if not set online version will be used from http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl]),
             [ XSL_STYLESHEET=$with_docbook ],
+            [ XSL_STYLESHEET=`find /usr/share/{xml,sgml} -path '*/manpages/docbook.xsl'` ],
```

Comment:
`AC_ARG_WITH` has only 4 parameters, this is incorrect.
The configure now fails with this option enabled (using autoconf 2.69 and automake 1.16.1).

Reply:
In addition, the output of `find /usr/share/{xml,sgml} -path '*/manpages/docbook.xsl'` is not one line for me:

```
/usr/share/xml/docbook/xsl-stylesheets-1.79.2-nons/manpages/docbook.xsl
/usr/share/xml/docbook/xsl-stylesheets-1.79.2/manpages/docbook.xsl
find: '/usr/share/sgml': No such file or directory
```